etheno/__main__.py

```diff
@@ -164,6 +150,8 @@ def main(argv=None):
         ETHENO.add_plugin(EventSummaryExportPlugin(args.export_summary))
 
     # First, see if we need to install Echidna:
+    '''
```

Comment:
I'd personally prefer removing unused code instead of leaving it commented out; it keeps the codebase cleaner and If someone needs it in the future, it can always be retrieved from git history. Or are you planning on restoring this feature shortly?

Reply:
No you're right, I'll remove it all unused code. I'm not sure if this stuff will be added back again.